index.bs

```diff
@@ -13396,6 +13396,26 @@ Extensions to any other aspect of the IDL language are
 strongly discouraged.
 
 
+<h2 id="legacy-constructs">Legacy constructs</h4>
+
+<i>This section is informative.</i>
+
+Legacy WebIDL constructs exist only so that
+legacy Web platform features can be specified.
+They are generally prefixed with the "<code>Legacy</code>" string.
+It is strongly discouraged to use legacy WebIDL constructs in specifications
+unless required to specify the behavior of legacy Web platform features,
+or for consistency with such features.
+Editors who wish to use legacy WebIDL constructs are strongly advised to discuss this
+by <a href="https://github.com/heycam/webidl/issues/new?title=Intent%20to%20use%20a%20legacy%20WebIDL%20construct">filing an issue</a>
+before proceeding.
+
+Legacy WebIDL constructs are not deprecated unless they are specifically marked as such.
```

Comment:
I don't think this paragraph is helpful because we don't have any notion of "deprecated" on the web platform. So this is just confusing.

If you want to talk about "removal" that's a separate story.

Reply:
That's a good point.